numbat/src/bytecode_interpreter.rs

```diff
@@ -109,6 +109,32 @@ impl BytecodeInterpreter {
                 self.compile_expression(lhs)?;
                 self.vm.add_op(Op::LogicalNeg);
             }
+            Expression::BinaryOperator(
+                _span,
+                op @ (BinaryOperator::LogicalOr | BinaryOperator::LogicalAnd),
+                lhs,
+                rhs,
+                _type,
+            ) => {
+                self.compile_expression(lhs)?;
+
+                let jump_offset = self.vm.current_offset() + 1; // +1 for the opcode
```

Comment:
How come you add 1 here? Can't you create it after adding the opcode to avoid the hardcoded +1?